gp-populate-anything/gppa-acf-repeater-mapper.php

```diff
@@ -36,7 +36,8 @@
 
 	foreach ( $field->{'gppa-choices-templates'} as $template => $key ) {
 		// Look for ACF repeater meta: repeater_name_0_subfield_name
-		if ( preg_match( '/^meta_(.*?)_([^_]+)_([^_]+)$/', $key, $matches ) ) {
+		// Known Limitation: We cannot have a number on the repeater field name.
```

Comment:
Thanks for the comment—can you expand on this and add an example of what will work and what won't work?

Reply:
I'd also add this known limitation in the instructions so users can catch this on their own. Also, I mentioned that we could suggest using "a, b, c" if they need to use an enumerator in their field names.